strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/services/impl/ConfigurationManagementServiceImplTest.java

```diff
@@ -140,9 +144,10 @@ public void testGetGroupRepositories()
 
     @Test
     @ExtendWith(RepositoryManagementTestExecutionListener.class)
-    public void testGetGroupRepositoriesContainingRepository(@NullRepository(repositoryId = REPOSITORY_RELEASES_1) Repository releases1,
-                                                             @TestRepository.Group(repositories = REPOSITORY_RELEASES_1)
-                                                             @NullRepository(repositoryId = REPOSITORY_GROUP_1) Repository releasesGroup)
+    public void testGetGroupRepositoriesContainingRepository(
+            @NullRepository(repositoryId = REPOSITORY_RELEASES_1) Repository releases1,
+            @TestRepository.Group(repositories = REPOSITORY_RELEASES_1)
+            @NullRepository(repositoryId = REPOSITORY_GROUP_1) Repository releasesGroup)
```

Comment:
Could you please preserve the original formatting? (Here and below). Thanks!

Reply:
Copy that!